core/commonMain/src/implementations/immutableMap/PersistentHashMapBuilderContentViews.kt

```diff
@@ -21,16 +31,14 @@ internal class PersistentHashMapBuilderEntries<K, V>(private val builder: Persis
 
     override fun remove(element: MutableMap.MutableEntry<K, V>): Boolean {
```

Comment:
TODO: Check if this function also prevents `Map.Entry`

Reply:
Done: `BuilderEntries.remove` also needs the workaround to be able to remove `Map.Entry` elements